builtin/credential/aws-ec2/path_config_certificate.go

```diff
@@ -64,14 +88,23 @@ func pathConfigCertificate(b *backend) *framework.Path {
 	return &framework.Path{
 		Pattern: "config/certificate/" + framework.GenericNameRegex("cert_name"),
 		Fields: map[string]*framework.FieldSchema{
-			"cert_name": &framework.FieldSchema{
+			"cert_name": {
 				Type:        framework.TypeString,
 				Description: "Name of the certificate.",
 			},
-			"aws_public_cert": &framework.FieldSchema{
+			"aws_public_cert": {
 				Type:        framework.TypeString,
 				Description: "AWS Public cert required to verify PKCS7 signature of the EC2 instance metadata.",
 			},
+			"type": {
```

Comment:
It makes sense. Added the default value here, updated the description about the default value. Marked this field as an optional field in the website and updated the description there as well.